### ISSUE TITLE
Server Side Rendering Error

### ISSUE THREAD

**Issue by coolsam726** [MattJeanes/ngx-parallax#9]
My Parallax Component is working great when I run my app in the dev server. However, my angular application is universal and supports server-side rendering. When I build the app with with server side rendering (with node server) I encounter the following error when I try running the server:

```
ReferenceError: HTMLElement is not defined
```
Obviously the error results from the fact that Server-side rendering does not support DOM elements, and somehow ngx-parallax is using HTMLElement somewhere which is a DOM (can only be run on the browser). Is @MattJeanes any possibility of making the module universal-compatible?

**Comment by AmyJeanes** [comment]
Hi Sam, if you take a look at the relatively simple code in this repository you can see how it works, and yes it is using HTMLElement to both read and set values on the DOM.

Obviously parallax is a visual only effect so it may be an idea to essentially disable it in a server side rendering context until the client takes over - though I'm not that familiar with Angular Universal and wouldn't know exactly how to make that change.

I'd be happy to accept any PRs to resolve this issue. As mentioned in the repository description I'm only the maintainer of this version of angular parallax, I didn't write the original.

**Comment by lazysergey** [comment]
@MattJeanes you might want to perform the simple isPlatformBrowser() check, so it would not call anything DOM-related and will not fire the error on server side

**Comment by AmyJeanes** [comment]
Yeah I see what you mean - although it wouldn't actually do anything until client takeover it would cause issues if you tried to load it on the server or only load on the client.

Happy to accept a PR that makes this universal compatible.

**Comment by fareshan** [comment]
Hi
Could you fix this issue ?

**Comment by marvinrusinek** [comment]
I am also running into the same ReferenceError message when trying to use SSR with Angular Universal. I'm currently using ngx-parallax version 3.0.0-rc.2 because somehow I couldn't get the later versions to work within Angular 7 with the imports. Please could you keep us posted with any news. Thanks!
